app/code/community/Kirchbergerknorr/UrlCacheControl/Mage/Core/Model/Design/Package.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/**
+ * @author      Nick Dilßner <nd@kirchbergerknorr.de>
+ * @copyright   Copyright (c) 2018 kirchbergerknorr GmbH (http://www.kirchbergerknorr.de)
+ * @license     http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class Kirchbergerknorr_UrlCacheControl_Mage_Core_Model_Design_Package extends Mage_Core_Model_Design_Package
```

Comment:
When overriding a Magento core class, we've always preferred to use the same class path as the original class within the module, i.e. "UrlCacheControl/Model/..." here.

Reply:
Kirchbergerknorr_UrlCacheControl_**Mage_Core_Model_Design_Package** != Mage_Core_Model_Design_Package?

Or did You mean that you only want: `Design_Package`?

You know that you mix your Models with Overwritten Core Models!? That is no good structure and also You see at the very first tier where to find overwritten models. As Example, if i need to overwrite the Product Block and the Product model, you have to folders to search for, with this naming, you have only `Mage/Catalog/*`

But ok, in this case it is a extened Version so it is a new Model and so you have better arguments